src/web/handlers/FlingGestureHandler.ts

```diff
@@ -120,18 +127,26 @@ export default class FlingGestureHandler extends GestureHandler {
     }
   }
 
-  protected onPointerMove(event: AdaptedEvent): void {
+  private pointerMove(event: AdaptedEvent): void {
```

Comment:
```suggestion
  private pointerMoveAction(event: AdaptedEvent): void {
```
I still find this name a bit confusing 😅